warehouse/core/src/main/java/datawave/edge/util/EdgeKey.java

```diff
@@ -44,6 +44,8 @@ public class EdgeKey {
 
     private static final Logger log = Logger.getLogger(EdgeKey.class);
 
+    private static final String CANT_ENCODE_UNKNOWN_ERROR_MSG = "Can't encode unknown edge key format.";
```

Comment:
`UNKNOWN_EDGE_KEY_FORMAT_MSG` seems more precise

Reply:
Sure, changed the name.